src/controllers/FrameController.ts

```diff
@@ -313,4 +313,16 @@ export class FrameController {
         const res = await this.#editorAPI;
         return res.removeFrame(frameId).then((result) => getEditorResponseData<null>(result));
     };
+
+    /**
+     * This method will assign an image from the mediaConnector to the correct imageFrame
+     * @param frameId The ID of the imageFrame where an image needs to be assigned to
```

Comment:
Would be good to repeat in the @param that this can only work when `frameId` is of type `imageFrame`